pkg/controller/migmigration/task.go

```diff
@@ -406,7 +417,7 @@ func (t *Task) Run() error {
 				}
 			}
 		}
-		t.Requeue = NoReQ
+		t.Requeue = PollReQ
```

Comment:
@sseago this is me changing NoReQ for DIM. Let me know if you want me to change this.